plugins/woocommerce-blocks/assets/js/blocks/checkout/styles/style.scss

```diff
@@ -43,6 +43,11 @@
 	flex-wrap: wrap;
 	margin: 0 auto $gap;
 	position: relative;
+
+	&:has(> :last-child.wp-block-woocommerce-checkout-fields-block) {
```

Comment:
Same as [this comment](https://github.com/woocommerce/woocommerce/pull/49649/files#r1683059922).